tests/integration/test_solr_histopheno.py

```diff
@@ -9,4 +9,5 @@ def test_histopheno():
         total += k.count
         
     assert hp
-    assert total == hp.total
\ No newline at end of file
+    assert hp.items[0].id == "HP:0000924"
+    # TODO:  Assert something about the total
```

Comment:
I don't think we actually need to assert anything about the total, it's really just about the counts per HP id

Reply:
cool, i'll just remove the comment and leave the test as is until we decide what we actually wanna test!